### PR TITLE
Jody/refactor

### PR DESCRIPTION
**Pull request by JodyZ0203**
1. switch to the new healthchecks api endpoint
2. add info tag for better readibility

Screen Shot 2023-02-07 at 11 07 02 PM

### CODE REVIEW

**Review by ben-z — COMMENTED** [review]
Looks really good overall! Finally we are removing these hard-codes

I just realized a few things:
- bastion's check is not here. Perhaps we need to add it in the host-config.yml in the infra-config repo
- We still hard-code bastion, but perhaps that's okay. I'm thinking about how we can display instructions clearly when we have multiple bastions.
- Checks have different name formats, e.g. `Report NVIDIA driver status` vs `cluster-connection`. I wonder if we want to consolidate them.

**Comment by ben-z** [comment]
Oh also, minor nitpick: currently the entire pane is clickable. Do we want to only make the information button clickable?

**Comment by JodyZ0203** [comment]
> Oh also, minor nitpick: currently the entire pane is clickable. Do we want to only make the information button clickable?

Yes we want to make the entire pane clickable because adding the information button is to let people know that there would be pop up modal.

File: pages/index.js [hunk 1]
```diff
@@ -182,32 +151,31 @@ export default function Home() {
...
+          <p style={{ color: 'white', marginRight: 10, marginLeft: 4 }}>
+            No Issues
+          </p>
+          <XIcon className="rounded-full h-5 w-5 text-white bg-red-600" />{' '}
+          <p style={{ color: 'white', marginRight: 10, marginLeft: 4 }}>
+            Outage
+          </p>
+          <PauseIcon className="rounded-full h-5 w-5 text-white bg-gray-600" />
+          <p style={{ color: 'white', marginRight: 10, marginLeft: 4 }}>
+            Paused
```
[the remark this answers is not in the document]
Comment (JodyZ0203) [reply pages/index.js:165]: Can you elaborate more on Report NVIDIA driver status vs cluster-connection? Do they mean the same thing.

**Comment by JodyZ0203** [comment]
Screen Shot 2023-02-24 at 1 15 08 AM

**Comment by ben-z** [comment]
The preview is showing nothing for some reason:

**Comment by ben-z** [comment]
The preview is showing nothing for some reason: 
E0195C00-DFD9-4C98-895C-5F5DBE2FC514

**Comment by ben-z** [comment]
Can we link the PRs in the infra-config repo that’s required by this PR?

**Comment by JodyZ0203** [comment]
> The preview is showing nothing for some reason: E0195C00-DFD9-4C98-895C-5F5DBE2FC514

I think its because the public tag is not added, or need to rerun terraform

**Comment by JodyZ0203** [comment]
I provisioned healthchecks again, and this what I get now
Screen Shot 2023-03-08 at 12 47 30 AM

**Comment by ben-z** [comment]
Great! There are a few more things that I think we should polish:
1. The access instruction for bastion should be different than the instructions for other VMs the use bastion as a jump host: 
F9B03A21-9683-4714-B0EB-4BE89EDDF269
2. Can we change the descriptions of the checks to match the wording used on the current page? `SSH, GPU, Filesystem Capacity` (note the slightly different capitalization in filesystem capacity).
3. The ordering of the checks are inconsistent across VMs. Perhaps we can simply sort them per VM?

File: pages/index.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (ben-z) [reply pages/index.js:187]: > Can you elaborate more on Report NVIDIA driver status vs cluster-connection? Do they mean the same thing.

Discussed offline a while ago.

Can we simplify this to only include no issues and outage? Perhaps we can assume everything other than `down` is up.

Perhaps we can have a mapping between healthchecksio status and our up/down status, and link the docs: https://healthchecks.io/docs/api/#list-checks

File: pages/index.js [hunk 2]
```diff
@@ -1,76 +1,67 @@
...
   const [show, setShow] = useState(false);
   const handleClose = () => setShow(false);
   const handleShow = () => setShow(true);
-  const { data: checks, error: errorChecks } = useSWR('/v1/checks/', fetcher, {
+  const { data: checks, error: errorChecks } = useSWR('/v2/checks/', fetcher, {
     refreshInterval: 30000,
     refreshWhenHidden: true,
   });
-
+  const vmOrder = [
```
[the remark this answers is not in the document]
Comment (ben-z) [reply pages/index.js:24]: Perhaps we can sort the groups by the number of checks, then by name?

**Comment by ben-z** [comment]
@JodyZ0203 what's the status of this PR? We just hit our Healthchecks.io limit and we should get this in so we can delete the old checks

**Comment by ben-z** [comment]
Cleaning up legacy PRs

**Pull request closed without merging**